docs/reference/query-dsl/mlt-query.asciidoc

```diff
@@ -241,6 +241,13 @@ number of terms that must match.
 The syntax is the same as the <<query-dsl-minimum-should-match,minimum should match>>.
 (Defaults to `"30%"`).
 
+`fail_on_unsupported_field`::
+Controls whether the query should fail (throw an  exception) if any of the 
```

Comment:
nit: double-space

Reply:
Darn space bar sticks; thanks for the catch!